### DISCUSSION TITLE
TypeScript lint errors when starting new project using `npx create-next-app --typescript`

### DISCUSSION THREAD

**Discussion by matronator** [vercel/next.js#26718] [Help] [answered]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

12.18.2

### What browser are you using?

Safari

### What operating system are you using?

macOS

### How are you deploying your application?

irrelevant

### Describe the Bug

After creating a brand new project using `npx create-next-app --typescript` and opening it, the linter reports errors in pretty much every `.ts` file.

Screenshot:

<img width="1268" alt="Snímek obrazovky 2021-06-28 v 21 40 32" src="https://user-images.githubusercontent.com/5470780/123694321-79146200-d859-11eb-8d45-b9555d085235.png">

### Expected Behavior

Create a project without linter errors.

### To Reproduce

1. `npx create-next-app --typescript`
2. Open the created project folder (`my-app`) in VSCodium / VSCode
3. Open any `.ts` file
4. ???
5. ~~_Profit_~~ see the errors

**Comment by timneutkens** [comment, marked as answer]
Both `./node_modules/.bin/tsc --noEmit` and vscode work fine when following the reproduction steps. This is likely related to your instance of vscode not loading the tsconfig that was created or something like that, can you try reloading the vscode window?

I'll convert it to a help discussion given this is not reproducible based on the steps provided.

**Reply by matronator** [reply]
Restarting the IDE fixed it. Thank you! :)
